www/src/context/SettingsContext.tsx

```diff
@@ -44,12 +45,16 @@ function SettingsContextProvider({ children }) {
     }
     setSettings(prevState => ({ ...prevState, [key]: value }));
     global.localStorage.setItem('pgn__settings', JSON.stringify({ ...settings, [key]: value }));
-    global.analytics.track(`openedx.paragon.docs.settings.${key}.changed`, { [key]: value });
+    sendUserAnalyticsEvent(SETTINGS_EVENTS.CHANGED, { [key]: value });
```

Comment:
We should probably also update the value that we send in the event, right now it will be hard to query this event in the database (e.g. count of `theme` changes).

Instead of `{ [key]: value }` let's send `{ setting: key, value }`.